lib/aerospike/cluster/find_nodes_to_remove.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# Copyright 2018 Aerospike, Inc.
+#
+# Portions may be licensed to Aerospike, Inc. under one or more contributor
+# license agreements.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+module Aerospike
+  class Cluster
+    # Calculates which nodes that should be removed from the cluster
+    module FindNodesToRemove
+      class << self
+        def call(cluster, refresh_count)
```

Comment:
It looks like we don't actually need the refresh counter - the function just cares whether the counter is zero or not, i.e. whether any node was refreshed successfully or not. So maybe we should just pass `peers.refresh_count.zero?` as a boolean instead? But I'm also ok to keep it as it is - as the algorithm to "find nodes to remove" evolves, we might need the actual counter again in the future. (And also to not unnecessarily diverge from the Java implementation.)

Reply:
Ok, I'll leave it as is for now then, for consistency